fragments/1602268921084.misc

```diff
@@ -0,0 +1 @@
+Atualiza .nvmrc para versão 14.13.1, removendo o termo 'latest'.
```

Comment:
Could you write this changelog in english?

Reply:
Sorry! Done.